### PR TITLE
Update "Club Penguin Rewritten" to v4.0.0.

### PR DESCRIPTION
**Pull request by BartBoys3**
Version 4.0.0 adds status for help documentation, community posts, and the new minigame "Smoothie Smash". 
"cprewritten.net/" is now "Viewing Homepage".
image
"play.cprewritten.net/" is now "Viewing Main Menu".
image
for "community.cprewritten.net/":
image
for "community.cprewritten.net/blog/":
image
for "community.cprewritten.net/category/news/":
image
for "community.cprewritten.net/author/stu/":
image
for "community.cprewritten.net/2020/05/10/reviewed-by-you-hidden-rooms/":
image
for "community.cprewritten.net/legal-disclaimer/":
image
for "cprewritten.net/help/":
image
for "cprewritten.net/help/articles/your_penguin.html":
image
for the new minigame "Smoothie Smash":
image
I think that's everything.

### CODE REVIEW

File: websites/C/Club Penguin Rewritten/presence.ts [hunk 1]
```diff
@@ -381,7 +381,76 @@ presence.on("UpdateData", async () => {
...
+          presenceData.state = "Safety";
+        } else if (document.location.pathname == "/help/articles/other_penguins.html") {
+          presenceData.state = "Other Penguins";
+        } else if (document.location.pathname == "/help/articles/puffles.html") {
+          presenceData.state = "Puffles";
+        } else if (document.location.pathname == "/help/articles/communicating.html") {
+          presenceData.state = "Communicating";
+        } else if (document.location.pathname == "/help/articles/tutorial.html") {
+          presenceData.state = "Tutorial";
+        }
```
Comment (Timeraa) [inline websites/C/Club Penguin Rewritten/presence.ts:450]: Just a suggestion but I would recommend using switch case statements

Comment (BartBoys3) [reply websites/C/Club Penguin Rewritten/presence.ts:450]: I'll read up on that.

Comment (BartBoys3) [reply websites/C/Club Penguin Rewritten/presence.ts]: how's this? `else if (document.location.host == "cprewritten.net") {
      if (document.location.pathname == "/") {
        presenceData.details = "Viewing Homepage";
      } else if (document.location.pathname.includes("/help/")) {
        presenceData.details = "Getting Help";
        if (document.location.pathname.includes(".html")) {
          const uncaptializedState = document.location.pathname.replace("/help/articles/", "").replace(".html", "");
          let state = "";
          const words = uncaptializedState.split("_");
          for (let i = 0; i < words.length; i++) {
            if (i == 0) {
              state = words[i].charAt(0).toUpperCase() + words[i].slice(1);
            } else {
              state = state + " " + words[i].charAt(0).toUpperCase() + words[i].slice(1);
            }
          } presenceData.state = state;
        }
      }
    }`

**Comment by BartBoys3** [comment]
Umm... about the deep scan errors, when I investigate them, they show files from another presence.

**Comment by BartBoys3** [comment]
This is the bit that was changed 
image
image

**Comment by doomlerd** [comment]
Update your fork btw

**Comment by BartBoys3** [comment]
I think it's not merging cause tim requested a change

**Pull request merged**